jvm/src/main/kotlin/app/cash/trifle/Trifle.kt

```diff
@@ -15,4 +29,70 @@ object Trifle {
     EndEntityDelegate by delegate {
     constructor(keysetHandle: KeysetHandle) : this(TinkDelegate(keysetHandle))
   }
+
+  fun SignedData.verify(certAnchor: ByteArray): Boolean {
+    val TRIFLE_SIGNING_MESSAGE_VERSION: Int = 0
+
+    if (certAnchor == null || certAnchor.isEmpty()) {
```

Comment:
this can be replaced with a `check() {...}` statement. also certAnchor is not nullable, so is the first check necessary?

Reply:
I wasnt sure if it's non nullable. But will remove the initial check.